measure.rb

```diff
@@ -1486,7 +1486,9 @@ def self.add_foundation_walls_slabs(runner, model, building, spaces)
         assembly_r = fnd_wall_values[:insulation_assembly_r_value]
         if assembly_r.nil?
           concrete_thick_in = fnd_wall_values[:thickness]
-          assembly_r = fnd_wall_values[:insulation_r_value] + Material.Concrete(concrete_thick_in).rvalue + Material.GypsumWall(drywall_thick_in).rvalue + film_r
+          int_r = fnd_wall_values[:interior_layer_r_value].nil? ? 0.0 : fnd_wall_values[:interior_layer_r_value]
+          ext_r = fnd_wall_values[:exterior_layer_r_value].nil? ? 0.0 : fnd_wall_values[:exterior_layer_r_value]
```

Comment:
Seems like we can simplify these lines to, e.g., `int_r = fnd_wall_values[:interior_layer_r_value]` because the layer R-values are now required when assembly R-value is not provided.